### DISCUSSION TITLE
MNIST TPU

### DISCUSSION THREAD

**Discussion by kamalkraj** [google/flax#1679] [Q&A] [answered]
Hi,

When I am running this [mnist](https://github.com/google/flax/tree/main/examples/mnist/) on V3-8 TPU-VM.

Will this model get trained on TPU core 0 or all the TPU cores(8)?

**Comment by jheek** [comment, marked as answer]
The mnist example uses `jit` rather than `pmap` so it will only train on a single core indeed.
